source/dotnet/Library/AdaptiveCards.Rendering.Wpf/Helpers/SelectActionHelper.cs

```diff
@@ -34,19 +34,7 @@ public static FrameworkElement RenderSelectAction(this AdaptiveRenderContext con
                 uiButton.Content = uiElement;
                 uiButton.Style = context.GetStyle("Adaptive.Action.Tap");
 
-                // Handle ShowCard
-                if (selectAction is AdaptiveShowCardAction showCardAction)
-                {
-                    var actionsConfig = context.Config.Actions;
-                    bool isInline = (actionsConfig.ShowCard.ActionMode == ShowCardActionMode.Inline);
-                    if (isInline && context.CardDepth == 1)
-                    {
-                        FrameworkElement uiShowCardContainer = showCardAction.CreateShowCard(context, actionsConfig);
-
-                        // Add to the list of show cards in context
-                        context.ActionShowCards.Add(new Tuple<FrameworkElement, Button>(uiShowCardContainer, uiButton));
-                    }
-                }
+                // SelectAction doesn't allow showCard actions
```

Comment:
>llow showCard actions [](start = 41, length = 21)

Does this fact (that selectaction doesnt support showCard action) need to be propagated up (Eg via an exception ) to the caller instead of silently ignoring? Also is this a breaking change?

Reply:
I'm following the model in UWP where I issue a warning for trying to make the select action a showcard action

---
In reply to: [272768066](https://github.com/Microsoft/AdaptiveCards/pull/2625#discussion_r272768066) [](ancestors = 272768066)